pkg/component/plutono/plutono.go

```diff
@@ -867,32 +867,15 @@ func (p *plutono) getEnvVar() []corev1.EnvVar {
 		{Name: "PL_AUTH_ANONYMOUS_ENABLED", Value: "true"},
 		{Name: "PL_USERS_VIEWERS_CAN_EDIT", Value: "true"},
 		{Name: "PL_DATE_FORMATS_DEFAULT_TIMEZONE", Value: "UTC"},
+		{Name: "PL_AUTH_BASIC_ENABLED", Value: "false"},
+		{Name: "PL_AUTH_DISABLE_LOGIN_FORM", Value: "true"},
+		{Name: "PL_AUTH_DISABLE_SIGNOUT_MENU", Value: "true"},
 	}
 
-	if p.values.IsGardenCluster {
-		envVar = append(envVar,
-			corev1.EnvVar{Name: "PL_AUTH_BASIC_ENABLED", Value: "false"},
-			corev1.EnvVar{Name: "PL_AUTH_DISABLE_LOGIN_FORM", Value: "false"},
-		)
-
-		return envVar
-	}
-
-	envVar = append(envVar,
-		corev1.EnvVar{Name: "PL_ALERTING_ENABLED", Value: "false"},
-		corev1.EnvVar{Name: "PL_SNAPSHOTS_EXTERNAL_ENABLED", Value: "false"},
-	)
-
-	if p.values.ClusterType == component.ClusterTypeSeed {
-		envVar = append(envVar,
-			corev1.EnvVar{Name: "PL_AUTH_BASIC_ENABLED", Value: "true"},
-			corev1.EnvVar{Name: "PL_AUTH_DISABLE_LOGIN_FORM", Value: "false"},
-		)
-	} else if p.values.ClusterType == component.ClusterTypeShoot {
+	if !p.values.IsGardenCluster {
 		envVar = append(envVar,
-			corev1.EnvVar{Name: "PL_AUTH_BASIC_ENABLED", Value: "false"},
-			corev1.EnvVar{Name: "PL_AUTH_DISABLE_LOGIN_FORM", Value: "true"},
-			corev1.EnvVar{Name: "PL_AUTH_DISABLE_SIGNOUT_MENU", Value: "true"},
+			corev1.EnvVar{Name: "PL_ALERTING_ENABLED", Value: "false"},
+			corev1.EnvVar{Name: "PL_SNAPSHOTS_EXTERNAL_ENABLED", Value: "false"},
```

Comment:
Thank you for simplifying the auth related environment variables above.

We could also simplify these values as well (if it is OK to do it in this PR): we don't need the alerting and the snapshot feature in the garden cluster's Plutono either. So we could remove the if condition that guards these lines. Thank you!

Reply:
Adapted in 58e7989113aa6dbef1b8344714bf9686b8a88734